Airwallex/Core/Sources/Internal/AWXCardValidator.m

```diff
@@ -79,16 +79,21 @@ + (instancetype)sharedCardValidator {
                            type:AWXBrandTypeUnknown],
 
         // American Express
-        [AWXBrand brandWithName:@"American Express"
+        [AWXBrand brandWithName:@"Amex"
                      rangeStart:@"34"
                        rangeEnd:@"34"
                          length:15
                            type:AWXBrandTypeAmex],
-        [AWXBrand brandWithName:@"American Express"
+        [AWXBrand brandWithName:@"Amex"
                      rangeStart:@"37"
                        rangeEnd:@"37"
                          length:15
                            type:AWXBrandTypeAmex],
+        [AWXBrand brandWithName:@"American Express"
```

Comment:
What about `American Express` starting with 37?

Reply:
Good question, this is something that I'm bit tentative to add, but I thought the above `Amex` has included number starts with both 34 and 37, which means both those card number can be matched to type `AWXBrandTypeAmex` already, and here I just wanna provide another brand name (full name for amex) that can be matched to that same type too, so we can even provide an empty number.

Reply:
Considering the consistency, I just also added 37 for that too.